lib/contributor.rb

```diff
@@ -0,0 +1,11 @@
+class Contributor
+  def initialize(contributor)
+    @contributor = contributor
+  end
+
+  def add
+    File.open('CONTRIBUTORS.md', 'a+') do |f|
+      f.puts "- [@#{@contributor}] (https://github.com/#{@contributor})"
+    end
+  end
+end
```

Comment:
I would suggest allowing pass a file to this class so we can better test it.
Also, moving `contributor` param to method `add` would allow us to reuse object in a batch in future.

```ruby
class Contributors
  def initialize(file_path = 'CONTRIBUTORS.md')
    @file_path = file_path
  end

  def add(contributor)
    File.open(@file_path) do |f|
      f.puts "- [@#{@contributor}] (https://github.com/#{@contributor})"
    end
  end
end
```

After this, you can use Ruby's Tempfile to test without having to worry about the original file.